requirements.txt

```diff
@@ -1,6 +1,35 @@
-Flask
-Flask-SQLAlchemy
-Flask-Script
-Flask-Bootstrap
-Flask-Migrate
-requests
+alembic==0.9.1
+appdirs==1.4.3
+astroid==1.5.2
+backports.functools-lru-cache==1.3
+click==6.7
+configparser==3.5.0
+dominate==2.3.1
+enum34==1.1.6
+flake8==3.3.0
+Flask==0.12.1
+Flask-Bootstrap==3.3.7.1
+Flask-Migrate==2.0.3
+Flask-Script==2.0.5
+Flask-SQLAlchemy==2.2
+isort==4.2.5
+itsdangerous==0.24
+Jinja2==2.9.6
+lazy-object-proxy==1.3.1
+Mako==1.0.6
+MarkupSafe==1.0
+mccabe==0.6.1
+packaging==16.8
+pep8==1.7.0
+pycodestyle==2.3.1
+pyflakes==1.5.0
```

Comment:
Split out the files for this into dev-requirements.txt and include requirements.txt inside it.